Leonardo/users relationship impl (#260)
desmos-labs/desmos: docs/developers/msgs/create-relationship.md

# `MsgCreateRelationship`

> This message allows you to create a mono directional relationship with a specified user.

Reviewer: ```suggestion
This message allows you to create a relationship between the signer and a specified user.
```

> Mono directional relationships are like the follow of today's social networks.

Reviewer: ```suggestion
```
